File: WebApp/Scripts/app/model/BG0008.js
Ext.define('WEBAPP.model.BG0008', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'AGEN_NO', type: 'string' }, //廠商代碼
        { name: 'AGEN_NAMEC', type: 'string' }, //廠商名稱
        { name: 'PO_NO', type: 'string' }, //訂單編號
        { name: 'MMCODE', type: 'string' }, //院內碼
        { name: 'MMNAME_C', type: 'string' }, //中文名稱
        { name: 'MMNAME_E', type: 'string' }, //英文名稱
        { name: 'M_PURUN', type: 'string' }, //計量單位
        { name: 'PO_QTY', type: 'string' }, //訂單數量
        { name: 'PO_PRICE', type: 'string' }, //單價
        { name: 'DELI_QTY', type: 'string' }, //進貨量
        { name: 'NOIN_QTY', type: 'string' }, //未進貨量
    ]

});

File: WebApp/Models/BG/BG0008.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class BG0008 : JCLib.Mvc.BaseModel
    {
        public string AGEN_NO{ get; set; } //廠商代碼
        public string AGEN_NAMEC{ get; set; } //廠商名稱
        public string PO_NO{ get; set; } //訂單編號
        public string MMCODE{ get; set; } //院內碼
        public string MMNAME_C{ get; set; } //中文名稱
        public string MMNAME_E{ get; set; } //英文名稱
        public string M_PURUN { get; set; } //計量單位
        public string PO_QTY { get; set; } //訂單數量
        public string PO_PRICE{ get; set; } //單價
        public string DELI_QTY{ get; set; } //進貨量
        public string NOIN_QTY{ get; set; } //未進貨量
        public string AMOUNT { get; set; } //差異量
        public string AGEN_TEL { get; set; } //電話
        public string MAT_CLASS { get; set; } //物料類別
        public string DATA_ARR { get; set; } //DATA串連

        
    }
}